apps/i18n/applab/en_us.json

```diff
@@ -488,7 +488,6 @@
   "makerSetupInstallSerialConnector": "Please install the [Code.org Serial Connector Chrome App]({webstoreURL}).",
   "makerSetupInstallFirmataCP": "You should make sure it has the right firmware sketch installed. You can install the Firmata sketch [for Circuit Playground Express boards with these instructions]({firmataURLExpress}) and [for Circuit Playground Classic boards with these instructions]({firmataURLClassic}).",
   "makerSetupInstallFirmataMB": "You can update your micro:bit by clicking on the \"Update micro:bit\" button below. After you press the button, a window should pop up with a row that includes \"micro:bit\". Select that row and then click the \"Connect\" button.",
-  "makerSetupInstallFirmataOther": "You should make sure it has the right firmware sketch installed. You can install the Firmata sketch [for Circuit Playground Express boards with these instructions]({firmataURLExpress}) and [for Circuit Playground Classic boards with these instructions]({firmataURLClassic}). You can install the Firmata sketch [for micro:bit with these instructions]({firmataURLMB})",
```

Comment:
> You can install the Firmata sketch [for micro:bit with these instructions]({firmataURLMB})",

Should this say Circuit Playground and have a different link?

Reply:
Sorry! I initially placed this comment in the wrong place:
See https://github.com/code-dot-org/code-dot-org/pull/55616/files#r1445283319

This string is being removed because we filter ports so that we only accept serial devices if they're either micro:bit, circuit playground express, or circuit playground classic.

Reply:
Oh hah - it's a red line, not a green line. That makes sense =)